server/server.go

```diff
@@ -350,6 +392,33 @@ func (s *Server) concurrentRequest(ctx context.Context, conf map[string]lookout.
 	return comments, nil
 }
 
+func mergeConfigs(global, local map[string]lookout.AnalyzerConfig) map[string]lookout.AnalyzerConfig {
+	if local == nil {
+		return global
+	}
+
+	if global == nil {
+		return local
+	}
+
+	merged := make(map[string]lookout.AnalyzerConfig)
+	for k, v := range global {
+		merged[k] = v
+	}
+
+	for k, v := range local {
+		if globalV, ok := merged[k]; ok {
```

Comment:
I think we're not correctly giving priority to local configuration here. If `k` is already present in `merged` we're just overriding the `settings` of the global configuration with the local ones. But what about if my local configuration has different `Addr` or `Disabled` or `Feedback`?

This should be:
```
		if globalV, ok := merged[k]; ok {
			v.Settings = mergeMaps(globalV.Settings, v.Settings)
			merged[k] = v
			continue
		}
```

Reply:
The `Addr` or `Feedback` fields are readed from the server conf instead of the org or repo configurations ([see docs](https://docs.sourced.tech/lookout/configuration#lookout-yml) about repo `.lookout.yml`).

`Disabled` field from org+repo configurations can only disable the analyzer because if it's already disabled by the server conf, this field from org+repo configuration is ignored by `lookoutd` ([sourcecode](https://github.com/src-d/lookout/blob/master/server/server.go#L292)).

Reply:
Ok, thanks for the clarification @dpordomingo.